test/hierarchical/test_ordinal.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+from typing import Dict, List
+
+import numpy as np
+import petab
+import pytest
+
+import pypesto
+import pypesto.logging
+import pypesto.optimize
+import pypesto.petab
+
+inner_solver_options = [
```

Comment:
this can surely be reduced (+ readability improved) with e.g. a few list comprehensions ...

Reply:
Reduced :D 

```
inner_solver_options = [
    [
        {
            key: value
            for key, value in zip(
                ['method', 'reparameterized', 'intervalConstraints'],
                [method, reparameterized, intervalConstraints],
            )
        }
        for method, reparameterized in zip(
            ['standard', 'reduced', 'reduced'], [False, False, True]
        )
    ]
    for intervalConstraints in ['max', 'max-min']
]
```